src/client/localplayer.cpp

```diff
@@ -250,7 +247,7 @@ void LocalPlayer::move(f32 dtime, Environment *env, f32 pos_max_d,
 	/*
 		Check if player is in liquid (the stable value)
 	*/
-	pp = floatToInt(position + v3f(0,0,0), BS);
+	pp = floatToInt(position + v3f(), BS);
```

Comment:
What's this?

Reply:
The default constructor of a `v3f` initialises all three components to 0. I assume this might be confusing to those who read the code, and if required I can make it explicit again, by changing it to `v3f(0.0f)` (single value constructor).

Reply:
Adding a null vector is pointless functional-wise, but it's okay to keep it explicit.

Reply:
Alright, I'll make it explicit by using the single value constructor. Thanks